### DISCUSSION TITLE
Question about perplexity api with openai-translate

### DISCUSSION THREAD

**Discussion by Adnios** [nextai-translator/nextai-translator#1365] [Q&A]
During past month, I was using perplexity api in openai-translate. And it works great. Below is my setting.

![image](https://github.com/openai-translator/openai-translator/assets/41060790/438f3392-bce9-4bf9-93ec-a43beb773c32)

But today, I hit errors "After the (optional) system message(s), user and assistant roles should be alternating."

![image](https://github.com/openai-translator/openai-translator/assets/41060790/cda0e360-1a9d-4e3a-8a3e-aad7332a3802)

Any insight? Thanks in advance!

**Comment by yetone** [comment]
新版本已修复
